lib/location/method_gnss.c

```diff
@@ -202,8 +244,15 @@ void method_gnss_lte_ind_handler(const struct lte_lc_evt *const evt)
 #if defined(CONFIG_NRF_CLOUD_MQTT)
 static void method_gnss_agps_request_work_fn(struct k_work *item)
 {
-	int err = nrf_cloud_agps_request(&agps_request);
+	int err;
 
+	err = check_modem_supports_lte();
+	if (err) {
+		LOG_ERR("nRF Cloud A-GPS request failed");
```

Comment:
You could add the reason (LTE disabled/not supported) in the error message.

Reply:
I will do this for the call to `check_modem_supports_gnss` as well